salesforce-oms/1.1/modules/ROOT/pages/anypoint-partner-manager-setup-guide.adoc

```diff
@@ -26,7 +26,7 @@ To run the collection, you must set the following variables:
 
 |env_identifier_ref-id_id |The ID of the identifier type Reference ID. This is static across all the organizations.
 
-|http_endpoint_type_id |The ID of the endpoint type HTTP.
+|http_endpoint_type_id |The `endpointTypeId` of an HTTPS endpoint type. This is static across all the organizations. Note that you can find the `endpointTypeId` value by opening the details of an endpoint in Partner Manager, and then running a request from the *Network* tab in the developer view of your browser. The value is available from the *Preview* tab.
```

Comment:
What does "This is static across all the organizations" mean?

Reply:
once you get the value it will remain same for all the environments in side an org